src/System.Private.CoreLib/shared/System/Globalization/CompareInfo.cs

```diff
@@ -934,20 +934,20 @@ public unsafe virtual int IndexOf(string source, string value, int startIndex, i
             return IndexOfCore(source, value, startIndex, count, options, null);
         }
 
-        internal int IndexOfOrdinal(ReadOnlySpan<char> source, ReadOnlySpan<char> value, bool ignoreCase)
+        internal int IndexOfOrdinal(ReadOnlySpan<char> source, ReadOnlySpan<char> value, bool ignoreCase, bool fromBeginning = true)
         {
             Debug.Assert(!_invariantMode);
             Debug.Assert(!source.IsEmpty);
             Debug.Assert(!value.IsEmpty);
-            return IndexOfOrdinalCore(source, value, ignoreCase);
+            return IndexOfOrdinalCore(source, value, ignoreCase, fromBeginning);
         }
 
-        internal unsafe int IndexOf(ReadOnlySpan<char> source, ReadOnlySpan<char> value, CompareOptions options)
+        internal unsafe int IndexOf(ReadOnlySpan<char> source, ReadOnlySpan<char> value, CompareOptions options, bool fromBeginning = true)
```

Comment:
>IndexOf(ReadOnl [](start = 28, length = 15)

There is a plan to expose IndexOf and LastIndexOf from CompareInfo too. I suggest to have these methods without the fromBeginning param and just let the IndexOfCore take the parameter.